pkg/operator/clustermemberremovalcontroller/clustermemberremovalcontroller.go

```diff
@@ -449,3 +479,14 @@ func hasInternalIP(machine *machinev1beta1.Machine, memberInternalIP string) boo
 	}
 	return false
 }
+
+func removeBootstrapMember(members []*etcdserverpb.Member) []*etcdserverpb.Member {
```

Comment:
Name is actually very confusing, it seems like it expects from this function to actually remove bootstrap and not just filter it from list

Reply:
done 👍🏽